_pages/about-us/teams/acqstack.md

```diff
@@ -113,7 +113,7 @@ These are a couple of suggested reading materials meant to give you a better sen
 
 #### 18F is here for you
 * **Ask for help.** Know there is a powerful ecosystem of 18Fers who are on-call to help with any areas where you need more assistance, whether it’s design, security practices, or product management best practices. We see new things every day, but it’s unlikely that someone hasn’t seen something like what you’re experiencing before and can offer a hand, ear, or whatever other body part you may need given the circumstances.
-* **Not everyone is ready for the 18F way.** We have a certain point of view when it comes to agile procurement and modular contracting.  It doesn’t work for all partners in every situation.  It’s ok to recognize that this may not be the right fit. What’s worse would be to force it onto someone not ready and see them stumble in [very predictable ways](https://18f.gsa.gov/2018/09/27/antipatterns-in-agile-contracting/). This wouldn’t normally be a problem but you may encounter folks who say something like, “Yea we tried that Agile thing once and it wasn’t for us, so we’re sticking with this.”
+* **Not everyone is ready for the 18F way on day one.** Our prinicples mirror agile software development and we are committed to modern procurement practices (ie. modular contracting). It’s ok to recognize that implementing new practices can be difficult and challenging.  What’s worse would be to force it onto someone not ready and see them stumble in [very predictable ways](https://18f.gsa.gov/2018/09/27/antipatterns-in-agile-contracting/). 
 ---
```

Comment:
FWIW, I think the last sentence (that we're striking) is probably worth pulling out and examining a bit more. I think it's pretty common, and it might be useful to note some strategies to have in your pocket when you meet resistance. That's just my opinion, though, and I'm approving this anyway.